### PR TITLE
fix: accessibility, SEO, typos, styling, favicons

### PR DESCRIPTION
**Pull request by Splaktar**
- add lang to html tag
- add robots.txt and sitemap.xml
- enable build-optimizer
- add Angular Material icon to footer
- fix typos
- use accent color on 2nd Get started button on homepage
- improve page title SEO
- better format copyright in footer
- add structured data, open graph, twitter to index
- preconnect and dns-prefetch fonts
- add apple-touch-icon links
- add site description, keywords, author, theme-color, Content-Type, etc.
- update favicons to Angular Material icons
  - support macOS touchbar and Windows 8/10 icons
- disable Angular CLI commonChunk
- refactor `main.scss` to `styles.scss` to be consistent with CLI

## Before
Screen Shot 2019-10-11 at 23 15 03
Screen Shot 2019-10-11 at 23 16 15

### Twitter
No preview.

### Structured Data
None found.

### Google
Screen Shot 2019-10-11 at 23 28 07

### Favicon checks
Screen Shot 2019-10-13 at 07 48 46

## After
Screen Shot 2019-10-11 at 23 15 18
Screen Shot 2019-10-11 at 23 15 51

### Twitter
Screen Shot 2019-10-11 at 23 19 56

### Structured Data
Screen Shot 2019-10-11 at 23 19 32

### Google preview
Screen Shot 2019-10-11 at 23 18 13

### Favicon checks
Screen Shot 2019-10-13 at 07 48 32

With assistance from https://realfavicongenerator.net

### CODE REVIEW

File: src/app/shared/footer/footer.html [hunk 1]
```diff
@@ -5,19 +5,23 @@
       <div class="footer-logo">
         <img class="docs-footer-angular-logo"
           src="../../../assets/img/homepage/angular-white-transparent.svg"
-          alt="angular">
+          alt="Angular icon">
```
Comment (crisbeto) [inline src/app/shared/footer/footer.html]: Shouldn't it be "Angular logo"?

File: src/app/shared/footer/footer.html [hunk 2]
```diff
@@ -5,19 +5,23 @@
...
         <span><a href="https://angular.io">Learn Angular</a></span>
       </div>
     </div>
 
     <div class="docs-footer-version">
+      <img class="angular-material-logo" src="../../../assets/img/angular-material-logo.svg"
+           alt="Angular Material icon">
```
Comment (crisbeto) [inline src/app/shared/footer/footer.html]: Should also say that it's the logo.

File: src/app/shared/footer/footer.scss [hunk 3]
```diff
@@ -30,7 +40,18 @@
   justify-content: flex-end;
   flex-direction: column;
   min-width: 225px;
-  text-align: center;
+  margin-top: 16px;
+
+  >div {
```
Comment (crisbeto) [inline src/app/shared/footer/footer.scss]: Needs a space after the >.

File: src/app/shared/footer/footer.html (hunk 1 above)
Comment (Splaktar) [reply src/app/shared/footer/footer.html]: Not based on https://hudsonvalleygraphics.com/is-it-a-logo-or-an-icon/ which aligns with my thinking that most logos include some typography.

https://blog.designcrowd.com/article/353/differences-between-logos-and-icons also mentions that icons are generally quadratic/square.

Here's the Angular logo: https://angular.io/assets/images/logos/angular/logo-nav@2x.png

**Pull request merged**